arci-ros/tests/util/msg_helper.rs

```diff
@@ -0,0 +1,27 @@
+#![cfg(target_os = "linux")]
+
+use std::sync::mpsc;
+
+/// # subscribe ROS message helper
+///
+/// using for inspect specific massage type.
+/// Message is displayed on screen and sent to ``mpsc receiver``
+///
+/// # Panic!
+///
+/// If subscriber can't be construct, this function is panic.
+///
+#[allow(dead_code)]
+pub(crate) fn subscribe_helper<T: rosrust::Message>(
```

Comment:
I'm afraid the name `subscribe_helper` does not indicate the role of this function properly.
How about `create_test_subscriber`, `create_subscriber_for_test` or something.

Reply:
Ok.